### PR TITLE
Add transport state to logging when loss detected

### PR DESCRIPTION
**Pull request by janaiyengar**

### CODE REVIEW

File: lib/quicly.c [hunk 1]
```diff
@@ -4621,6 +4636,9 @@ const char *quicly_event_attribute_names[] = {NULL,
                                               "cc-exit-recovery",
                                               "acked-packets",
                                               "acked-bytes",
+                                              "min_rtt",
+                                              "smoothed_rtt",
+                                              "latest_rtt",
```
Comment (kazuho) [inline lib/quicly.c]: Can we use `-` instead of `_`? I think that is the convention we have so far.

Other than that the PR looks good to me.

Comment (janaiyengar) [reply lib/quicly.c]: Done.

**Comment by janaiyengar** [comment]
@kazuho: any idea why the CI build is failing?

**Comment by kazuho** [comment]
@janaiyengar I suspect it's a timing issue; rerunning the CI resolved the problem. I've created and merged #105 so that we can see the actual log when it fails once more.

**Pull request merged**